giftcompare/giftcompare/urls.py

```diff
@@ -24,5 +24,4 @@
     path('', include('categories.urls')),
     path('', include('users.urls')),
     path('api-auth/', include('rest_framework.urls')),
-
```

Comment:
The global urls need to have your new loval user-registration url: `path('users/register/', UserRegistrationView.as_view(), name='user-register'),` otherwise we no longer can signup a user.

Reply:
I will fix this into my branch so we don't delay this merge.